app/bundles/PageBundle/Entity/Page.php

```diff
@@ -223,6 +234,12 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->nullable()
             ->build();
 
+        $builder->createOneToMany('submissions', Submission::class)
```

Comment:
I use it [here](https://github.com/mautic/mautic/blob/765f8ccc004408ce671b7e7268dd00e58d8fd59b/app/bundles/PageBundle/Views/Page/list.html.php#L120).
https://github.com/mautic/mautic/blob/765f8ccc004408ce671b7e7268dd00e58d8fd59b/app/bundles/PageBundle/Views/Page/list.html.php#L120

I didn't create a new assossiation. I just added a `submissions` property to the Page to be able to access the submissions that are related to a page. This might fix the problem. But Im not sure it is the optimal way.

Reply:
It is not used in this PR though, right? It does not allow us to remove the subquery in the `getEntities()` method, so I don't get what is the point adding it.

Reply:
It is used in this PR and fix the main problem. 

The subquery is only used to sort by column. I couldn't find any other way to do it. If we can do without the sorting, the subquery can be removed